bodhi/server/models.py

```diff
@@ -2390,18 +2389,7 @@ def edit(cls, request, data):
                 })
 
             # Add the pending_signing_tag to all new builds
-            for build in new_builds:
-                if up.from_tag:
-                    # this is a sidetag based update. use the sidetag pending signing tag
-                    side_tag_pending_signing = up.release.get_pending_signing_side_tag(up.from_tag)
-                    koji.tagBuild(side_tag_pending_signing, build)
-                elif up.release.pending_signing_tag:
-                    # Add the release's pending_signing_tag to all new builds
-                    koji.tagBuild(up.release.pending_signing_tag, build)
-                else:
-                    # EL6 doesn't have these, and that's okay...
-                    # We still warn in case the config gets messed up.
-                    log.warning('%s has no pending_signing_tag' % up.release.name)
+            tag_build(new_builds, up)
```

Comment:
Celery tasks are called using the `.delay()` method, please check out how it's done with the `handle_update()` task.